TESTS/mbedmicro-rtos-mbed/event_flags/main.cpp

```diff
@@ -17,47 +17,361 @@
 
 #include "mbed.h"
 #include "greentea-client/test_env.h"
-#include "rtos.h"
+#include "unity/unity.h"
+#include "utest/utest.h"
+
+using utest::v1::Case;
 
 #if defined(MBED_RTOS_SINGLE_THREAD)
   #error [NOT_SUPPORTED] test not supported
 #endif
 
-#define TEST_STACK_SIZE 512
+#define THREAD_STACK_SIZE   256 /* 512B stack on GCC_ARM compiler cause out of memory on some 16kB RAM boards */
```

Comment:
This stack is quite small. Which devices were failing?

Reply:
test fails on nucleo_f070rb with GCC_ARM compiler on develop profile with 512B and 400B
384B stack is ok, so could be increased a bit to  320B or 350B

Reply:
stack increased to 320B